geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/data/PartitionedRegionStatsUpdateTest.java

```diff
@@ -75,115 +75,166 @@ public void should_showIncreaseInDatastoreBytesInUse_givenStringValueSizeIncreas
     String LONG_APPEND_VALUE = String.valueOf(Integer.MAX_VALUE);
     jedis.set(STRING_KEY, "value");
 
-    long initialDataStoreBytesInUse =
+    long initialDataStoreBytesInUse1 =
         clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
 
     for (int i = 0; i < 1000; i++) {
       jedis.append(STRING_KEY, LONG_APPEND_VALUE);
     }
 
-    long finalDataStoreBytesInUse = clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
+    long finalDataStoreBytesInUse1 =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
 
-    assertThat(finalDataStoreBytesInUse).isGreaterThan(initialDataStoreBytesInUse);
+    assertThat(finalDataStoreBytesInUse1).isGreaterThan(initialDataStoreBytesInUse1);
     long server2finalDataStoreBytesInUse =
         clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server2);
-    assertThat(server2finalDataStoreBytesInUse).isEqualTo(finalDataStoreBytesInUse);
+    assertThat(server2finalDataStoreBytesInUse).isEqualTo(finalDataStoreBytesInUse1);
```

Comment:
Is there also supposed to be an `initialDataStoreBytesInUse2` in this test?

Reply:
No, this test has a different way of just making sure the final bytesInUse agree on server1 and server2. I just added "1" on the end of the values that come from server1. It did have the primary vs. secondary issues of other test methods because it does 1000 appends.